.github/workflows/ci_tests_legacy.yaml

```diff
@@ -58,8 +58,6 @@ jobs:
             channels:
               - conda-forge
               - nodefaults
-          cache-downloads: false
-          cache-environment: true
```

Comment:
This workflow is only scheduled to run weekly. We don't care about the speed of the workflow, so no need to cache the enviroment.